ontology/yaml/resources/HVAC/entity_types/BLR.yaml

```diff
@@ -271,6 +271,22 @@ BLR_SS_SWTC_WDPC:
   - SS
   - SWTC
   - WDPC
+  
+BLR_SS_HTRWC:
```

Comment:
See below comment. Would make sense to make this `BLR_SS_RWTC_HTRWC` for consistency.